runtime/layers/fpm-dev/Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG LAYER_IMAGE
-FROM $LAYER_IMAGE
+ARG PHP_VERSION
+FROM bref/php-$PHP_VERSION-fpm
```

Comment:
Those changes are mainly to help understand which image is actually used.

This is a minor change.